### DISCUSSION TITLE
About projects

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#86] [General]
I will create several subprojects to record the to-do list of different tasks. 

Because I realize it's messy to put them together. 

I then record the progress of the subprojects in [Brevis](https://github.com/orgs/abbrcodes/projects/1/).

I just realized this is a bit more complicated than Trello, but don't worry let me see how I do it.

**Comment by T1xx1** [comment]
> I will create several subprojects to record the to-do list of different tasks.
> 
> Because I realize it's messy to put them together.
> 
> I then record the progress of the subprojects in [Brevis](https://github.com/orgs/abbrcodes/projects/1/).
> 
> I just realized this is a bit more complicated than Trello, but don't worry let me see how I do it.

Ok don't worry

**Comment by T1xx1** [comment]
abbrcode/discussions#3

**Comment by oood** [comment]
I've just been testing and it seems like I can't easily move a task from one project to another.

So I figured they would have to be grouped together in Brevis so as not to create more confusion, I'd edit the intro for each task so we don't miss the details.

**Comment by oood** [comment]
UPDATE: Brevis is now linked below these repositories: 

https://github.com/abbrcodes/abbreviations-in-code and
https://github.com/abbrcodes/discussions

**Reply by T1xx1** [reply]
Great. Do you think we sho link every project under discussions anyway?

**Reply by oood** [reply]
That might be easier for others to see, and easier for contributors to get involved.

**Comment by oood** [comment]
https://github.com/orgs/abbrcodes/projects/1/views/1?pane=issue&itemId=22303620

Can you take a look at this? Is this format intuitive?

@T1xx1

**Reply by T1xx1** [reply]
Yes, should i close the issue now as we moved its content to the project?

**Reply by T1xx1** [reply]
I also labeled it in progress

**Reply by oood** [reply]
> Yes, should i close the issue now as we moved its content to the project?

Maybe, I'm moving this issue to the project altogether




> I also labeled it in progress

Thanks.


It's being edited, it doesn't link directly to the issues, the full URL needs to be copied, which is a bit of a hassle. 🤷‍♀️

**Reply by T1xx1** [reply]
> > Yes, should i close the issue now as we moved its content to the project?
> 
> Maybe, I'm moving this issue to the project altogether

Because it not an issue anymore but a task.
 
> It's being edited, it doesn't link directly to the issues, the full URL needs to be copied, which is a bit of a hassle. 🤷‍♀️

I know experiencing the same copying issues and prs URLs into discussion.

**Reply by oood** [reply]
> I know experiencing the same copying issues and prs URLs into discussion.

Maybe we should use sub-discussion modules in [that repository](https://github.com/abbrcodes/abbreviations-in-code)? 🤔

Discuss things related to the entire organization in the [discussions repository](https://github.com/abbrcodes/discussions)?

**Reply by T1xx1** [reply]
There's only one problem. With discussion, we "lose" the point of the conversation as I've already seen in some issues.

**Reply by oood** [reply]
> There's only one problem. With discussion, we "lose" the point of the conversation as I've already seen in some issues.

Yep
Sorry for digressing, hahaha

**Reply by T1xx1** [reply]
HAHAH not to worry. We can go around that problem starting with projects rather than issues. For now, we have to import things because they are somewhere else but I think things will eventually be easier after.

**Comment by T1xx1** [comment]
I'll start to do some tasks in [issue](https://github.com/abbrcodes/abbreviations-in-code/pull/50#issue-1609931125)

**Reply by oood** [reply]
> I'll start to do some tasks in [issue](https://github.com/abbrcodes/abbreviations-in-code/pull/50#issue-1609931125)

Thanks!

**Comment by oood** [comment]
Can we move the [task details](https://github.com/orgs/abbrcodes/projects/1/views/1?pane=issue&itemId=22303620) into the discussion?



Then just leave a name and a link to the discussion.

Because it is not convenient to discuss there.

**Reply by T1xx1** [reply]
I think I find out another way. Instead of copying the URL of the issue/pr and then paste in the project. If you go to the issue/pr in the right panel there's a project setting. I think you can issues/pr in the project from there. In this way, there's also a link to the project in the issue/pr.

**Reply by T1xx1** [reply]
Or even. Issues and prs can be grouped into milestones. You can link the milestone and have all the related project issues/prs there.

**Reply by oood** [reply]
> Or even. Issues and prs can be grouped into milestones. You can link the milestone and have all the related project issues/prs there.

Great! Everything started to be simple.

**Reply by T1xx1** [reply]
Told you HAHAHAH

**Comment by T1xx1** [comment]
@oood about Brevis.

I saw that to specify which part of the org the task was ref to you put [] before.
Isn't better to create a separate project named org.
If yes I'll move the tasks.

**Reply by oood** [reply]
Moving isn't easy, maybe a single project would be fine, but since copying and moving is inconvenient, then we have to keep an eye on multiple projects at the same time, which might not be suitable.

**Reply by oood** [reply]
I'm trying to make this list intuitive, the [tabs] are one way.

But with future growth, we may have to create new lists.

But why not now let's see how far we can go.

**Comment by T1xx1** [comment]
Can I del project 5?

**Reply by oood** [reply]
I think it's better to rewrite it, because the number will no longer be usable after deletion.

**Reply by oood** [reply]
I deleted the 2 3 4 of the test before, and then the numbers are no longer available.

**Reply by T1xx1** [reply]
Ok so I'll just leave it like that

**Reply by oood** [reply]
OK

**Comment by oood** [comment]
Hi guys.

I've just preliminarily sorted out our task list, [Brevis](https://github.com/orgs/abbrcodes/projects/1/views/1)

You can check it out [here](https://github.com/orgs/abbrcodes/projects/1/views/1).

There is currently a work in progress to [move the [ORG] tag to the new ORG list](https://github.com/orgs/abbrcodes/projects/1/views/1?pane=issue&itemId=22322485).



@T1xx1 @kisvegabor @philipeachille

**Comment by oood** [comment]
@T1xx1 

A suggestion, maybe we don't have to put the new abbreviation of the ABBRS repository into Brevis?

I think we make Brevis the hub for all to-do lists, so we can monitor all progress and tasks in one place.

But those small tasks we disperse into separate projects like ORG, ABBRS.

**Reply by T1xx1** [reply]
Ok. There's also a milestone with them so we can track those progress from there.

**Reply by oood** [reply]
sounds good

**Comment by T1xx1** [comment]
I've moved some tasks in the new org project (I haven't finished I will continue later).

@oood can I give a tip about tasks
I'm not an expert but I think it will make the project experience better.

There's no need to report the conversation we had about sth in the task. At the top add a link to the discussion/issue in the task and just write a plain list of the things we have to do. Nice and easy without content we don't need when we have to search for todos. In the same time if we need to revise the discussion there's the link with all the infos. Reporting only a part of the conversation is worthless.

**Reply by oood** [reply]
ok that would be easy

**Comment by oood** [comment]
@T1xx1 

Sorry to be really busy these days.

But I have rearranged my schedule so that I can focus on this project from 1pm to 2pm (UTC) every day except weekends.

Also, you can always message me via the IM (https://t.ly/textme ). there, I'll be online as much as possible, but I won't be chatting because there isn't much free time.

So let's do it, I'm back now, let me see what I can do to catch up.

**Reply by T1xx1** [reply]
> @T1xx1 
> 
> Sorry to be really busy these days.

Don't worry.

> But I have rearranged my schedule so that I can focus on this project from 1pm to 2pm (UTC) every day except weekends.

OK. Most of the times I'm doing it after dinner so we alt work.

> Also, you can always message me via the IM (https://t.ly/textme ). there, I'll be online as much as possible, but I won't be chatting because there isn't much free time.

OK but I think it's not necessary: everything I do it's here on GitHub so you can see all the process.


> So let's do it, I'm back now, let me see what I can do to catch up.

👍

**Reply by oood** [reply]
Ok, I'll keep an eye on the task list so I have something to do when I'm online.

I'm currently trying to rewrite the issue template, and I started [a new discussion](https://github.com/orgs/abbrcodes/discussions/9) to talk about it.
